src/task2.1.js

```diff
@@ -0,0 +1,73 @@
+const express = require('express');
+const app = express();
+const router = express.Router();
+const schema = require('./validations/user.schema');
+import { validateSchema } from './validations/validations';
+import { usersList } from './services/userDataService';
+
+const findUser = (userId) => {
+    return usersList.find((user) => user.id === userId);
+};
+
+app.listen(3000);
+app.use(express.json());
+
+router.get('/users/:id', (req, res) => {
```

Comment:
@lenon777, please separate server configuration and route handling. Try to keep your class/functions/files as simple as possible (KISS).